src/derived/col.js

```diff
@@ -5,9 +5,9 @@ import type { Props } from '../withBase'
 
 const defaults = {
   marginTop: 0,
-  marginRight: 0.5,
-  marginBottom: 1,
-  marginLeft: 0.5,
+  marginRight: 1.5,
+  marginBottom: 3,
+  marginLeft: 1.5,
```

Comment:
this looks rather arbitrary now but once we've moved to a CSS in JS and gutter size is also expressed as a configurable multiple of `base`, this can become `0 gutter_size/2 gutter_size` again